---a/net/minecraft/nbt/NBTTagString.java
+++b/net/minecraft/nbt/NBTTagString.java
@@ -4,78 +4,63 @@
 import java.io.DataOutput;
 import java.io.IOException;
 
-public class NBTTagString extends NBTBase
-{
+public class NBTTagString extends NBTBase {
+
     private String data;
     private static final String __OBFID = "CL_00001228";
 
-    public NBTTagString()
-    {
+    public NBTTagString() {
         this.data = "";
     }
 
-    public NBTTagString(String data)
-    {
+    public NBTTagString(String data) {
         this.data = data;
 
-        if (data == null)
-        {
+        if (data == null) {
             throw new IllegalArgumentException("Empty string not allowed");
         }
     }
 
-    void write(DataOutput output) throws IOException
-    {
+    void write(DataOutput output) throws IOException {
         output.writeUTF(this.data);
     }
 
-    void read(DataInput input, int depth, NBTSizeTracker sizeTracker) throws IOException
-    {
+    void read(DataInput input, int depth, NBTSizeTracker sizeTracker) throws IOException {
         this.data = input.readUTF();
-        sizeTracker.read((long)(16 * this.data.length()));
+        sizeTracker.read((long) (16 * this.data.length()));
     }
 
-    public byte getId()
-    {
-        return (byte)8;
+    public byte getId() {
+        return (byte) 8;
     }
 
-    public String toString()
-    {
+    public String toString() {
         return "\"" + this.data.replace("\"", "\\\"") + "\"";
     }
 
-    public NBTBase copy()
-    {
+    public NBTBase copy() {
         return new NBTTagString(this.data);
     }
 
-    public boolean hasNoTags()
-    {
+    public boolean hasNoTags() {
         return this.data.isEmpty();
     }
 
-    public boolean equals(Object p_equals_1_)
-    {
-        if (!super.equals(p_equals_1_))
-        {
+    public boolean equals(Object p_equals_1_) {
+        if (!super.equals(p_equals_1_)) {
             return false;
-        }
-        else
-        {
-            NBTTagString nbttagstring = (NBTTagString)p_equals_1_;
+        } else {
+            NBTTagString nbttagstring = (NBTTagString) p_equals_1_;
             return this.data == null && nbttagstring.data == null || this.data != null && this.data.equals(nbttagstring.data);
         }
     }
 
-    public int hashCode()
-    {
+    public int hashCode() {
         return super.hashCode() ^ this.data.hashCode();
     }
 
-    public String getString()
-    {
+    public String getString() {
         return this.data;
     }
-}
+}
